google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/DefaultCloudSdkService.java

```diff
@@ -38,6 +41,9 @@ public String getCloudSdkHomePath() {
 
   @Override
   public void setCloudSdkHomePath(String cloudSdkHomePath) {
-    propertiesComponent.setValue(CLOUD_SDK_PROPERTY_KEY, cloudSdkHomePath);
+    if (!StringUtil.isEmpty(cloudSdkHomePath)
+        && CloudSdkUtil.containsCloudSdkExecutable(cloudSdkHomePath)) {
+      propertiesComponent.setValue(CLOUD_SDK_PROPERTY_KEY, cloudSdkHomePath);
```

Comment:
Yea I was wavering on this point too. I can update it to allow them - with the caveat that certain UIs that take the SDK will require a valid path (e.g. the deployment config, but _not_ the google settings page).